blocks/nba/nba.js

```diff
@@ -0,0 +1,33 @@
+import { unwrapDivs } from '../../scripts/helpers.js';
+
+export default async function decorate(block) {
+  const textWrapper = document.createElement('div');
+  textWrapper.classList.add('nba-text-wrapper');
+
+  const headings = block.querySelectorAll('h1, h2, h3, h4, h5, h6');
+  [...headings].forEach((heading) => {
+    heading.classList.add('heading');
```

Comment:
I would add the h3 class here too - on the designs we have the h3 styles.
I understand that in this way we can give the author the option to choose the heading size, but there is a problem that using the heading types force the tag too. It can break the order of the headings in the document. IMO we have a design with a heading type which should always render in that way as it is on the design but with the tags that are selected by the author in the document.
We could give the option to select the rendered tag and the HTML tag for authors but I don't see a way to do that in the Google doc.

Reply:
Agreed, what you suggested makes sense. Have done these changes